package.json

```diff
@@ -7,11 +7,12 @@
     "@carbon/icons": "^10.5.0",
     "@commitlint/cli": "^8.2.0",
     "@commitlint/config-conventional": "^8.2.0",
-    "@storybook/addon-actions": "^5.1.11",
-    "@storybook/addon-info": "^5.1.11",
-    "@storybook/addon-knobs": "^5.1.11",
-    "@storybook/addon-storysource": "^5.1.11",
-    "@storybook/react": "^5.1.11",
+    "@storybook/addon-actions": "5.3.0-beta.15",
```

Comment:
we should toss a issue in to upgrade to `5.3.0` at some point https://github.com/storybookjs/storybook/releases/tag/v5.3.0-rc.0